src/main/kotlin/uk/co/craigbass/pratura/domain/Currency.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.domain
+
+import org.joda.money.*
+import org.joda.money.format.MoneyFormatterBuilder
+import java.math.BigDecimal
+import java.util.*
+
+class Currency(val currency: String, val country: String, val language: String) {
+  fun format(amount: BigDecimal): String {
+    val currencyUnit = CurrencyUnit.of(currency)
```

Comment:
this presumably can throw exceptions, I'd also pass in a CurrencyUnit rather than a String

Generally speaking I avoid doing loads of String => Type conversions in functions but push that as far to the edges as possible